pkgs/development/python-modules/git-filter-repo/default.nix

```
@@ -1,14 +1,18 @@
 { lib
 , buildPythonPackage
-, fetchpatch
+, fetchFromGitHub
 , fetchPypi
+, fetchpatch
+, installShellFiles
 , pythonOlder
 , setuptools-scm
+, writeScript
 }:
 
 buildPythonPackage rec {
   pname = "git-filter-repo";
   version = "2.38.0";
+  docs_version = "01ead411966a83dfcfb35f9d2e8a9f7f215eaa65";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
Could you please replace `format` with `pyproject` to avoid the legacy build?

> Note that the pyproject format falls back to using setuptools, so you can use pyproject = true even if the package only has a setup.py. 
When unset, the legacy setuptools hooks are used for backwards compatibility.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters